docs/index.md

```diff
@@ -0,0 +1,42 @@
+# Joinmarket documentation
+
+## Contents
+
+1. User guide material
+
+   a. [Installation guide (BUT use the install script as per README, first)](INSTALL.md)
+
+   b. [Main usage guide, covers wallet funding, creation, control, transactions etc.](./USAGE.md)
+
+   c. [Running a yield generator general basics](./YIELDGENERATOR.md)
+
+   d. [Running JoinmarketQt for coinjoins and wallet control](JOINMARKET-QT-GUIDE.md)
+
+   e. [Important info for takers - how to deal with commitments](./SOURCING-COMMITMENTS.md)
+
+   f. [Important info for makers - how to create and use fidelity bonds](./fidelity-bonds.md)
+
+   g. [How to run the tumbler script (or in Qt) for a good privacy effect](tumblerguide.md)
+
+   h. [How to run the orderbook script locally](orderbook.md)
+
+   i. [How to do payjoins with Joinmarket](./PAYJOIN.md)
+
+   j. [(Describes a feature that is not currently active - SNICKER)](SNICKER.md)
```

Comment:
nit - I think this way is better, "feature that is not currently active" is bad naming for any functionality. :)
```suggestion
   j. [(How to use SNICKER features (currently not active))](SNICKER.md)
```

Reply:
I actually genuinely don't understand what you mean (though I certainly don't object to the change); what is the difference between 'feature that is not currently active' and 'feature (not currently active)'?

Reply:
Emphasis on feature itself vs it being not active.

Reply:
OK, done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1155/commits/8a87f5227cbc2ce31e83a06241e35ec1985a0bf3